packages/p2p-v2/src/components/AdvertiserName/AdvertiserNameStats.tsx

```diff
@@ -39,12 +39,15 @@ const AdvertiserNameStats = ({ advertiserStats }: { advertiserStats: TAdvertiser
             <div>
                 {!isMyProfile && (
                     <div className='border-r-[1px] border-solid border-r-[#ededed]'>
-                        <OnlineStatusIcon isOnline={isOnline} isRelative size='0.8em' />
-                        <OnlineStatusLabel isOnline={isOnline} lastOnlineTime={lastOnlineTime} />
+                        <OnlineStatusIcon isOnline={!!isOnline} isRelative size='0.8em' />
+                        <OnlineStatusLabel
+                            isOnline={!!isOnline}
+                            lastOnlineTime={lastOnlineTime === null ? undefined : lastOnlineTime}
```

Comment:
because the type is `number | null | undefined` and the function in `OnlineStatusLabel`, `getLastOnlineLabel` prop `lastOnlineTime` can be undefined but doesn't take null. If undefined, it will return a default string, so we shouldn't do `lastOnlineTime ?? 0` to return a default value.